src/Command/Test.php

```diff
@@ -180,23 +185,22 @@ private function mail(OutputInterface $output, bool $multipart = false): void
             $this->sendMailPackage($output, $socket, "QUIT\r\n", '221 ');
 
             \socket_close($socket);
-
         } catch (\Throwable $e) {
             $this->logger->exception($e, 'Mail protocol error', important: true);
         }
     }
 
     private function sendMailPackage(
         OutputInterface $output,
-        Socket $socket,
+        \Socket $socket,
         string $content,
         string $expectedResponsePrefix,
     ): void {
         if ($content !== '') {
             \socket_write($socket, $content);
             // print green "hello" string in raw console markup
             $output->write(
-                '> ' . \str_replace(["\r", "\n"], ["\e[32m\\r\e[0m", "\e[32m\\n\e[0m"], $content),
```

Comment:
The previous linter added spaces here. Now this removes ones :D

Reply:
Per 2 rule adds spaces, PhpCsFixer rule removes, which is better?